_posts/2020-03-31-DevOpsDays-2019.md

```diff
@@ -0,0 +1,235 @@
+---
+layout: post
+authors: [johan_silkens]
+title: 'DevOpsDays 2019'
+image: /img/2019-11-28-devopsdays-2019/devopsdays-amsterdam-2019.png
+tags: [DevOps, Conference]
+category: Conference
+comments: true
+---
+
+> The 7th edition of DevOpsDays Amsterdam in 'Pakhuis de Zwijger' was my first experience with [DevOpsDays](https://devopsdays.org/){:target="_blank" rel="noopener noreferrer"}. 
+> I'd like to share some talks and my impressions about the conference, because sharing ideas worth spreading is what we aim for at JWorks.
+
+# Table of contents
+* [Observability for Emerging Infra: What Got You Here Won't Get You There, by Charity Majors](#observability-for-emerging-infra-what-got-you-here-wont-get-you-there-by-charity-majors)
+* [How Convenience is Killing Open Standards, by Bernd Erk](#how-convenience-is-killing-open-standards-by-bernd-erk)
+* [Come Listen to me, I'm a fraud, by Joep Piscaer](#come-listen-to-me-im-a-fraud-by-joep-piscaer)
+* [Fight, Flight or Freeze – Releasing Organizational Trauma, by Matty Stratton](#fight-flight-or-freeze--releasing-organizational-trauma-by-matty-stratton)
+
+## Observability for Emerging Infra: What Got You Here Won't Get You There, by Charity Majors
```

Comment:
Would be nice to have an image in the middle of the section. Just to break up the text a bit.

Reply:
I've added a Venn diagram somewhere in the middle that shows the relation beween metric, logging , .. etc and such. Since Observabilty has the advantages of all three :)